### ISSUE TITLE
Gradle provider: When using state change settings and switching from x.10 to x.12 or x.13 pact verification fails

### ISSUE THREAD

**Issue by peterlitvak** [DiUS/pact-jvm#113]
It fails with the following error:
```
No such property: statusCode for class: org.apache.http.impl.execchain.HttpResponseProxy
Possible solutions: statusCode, statusLine
```
The state change is setup like so:
```
stateChange = 'http://localhost:8080/dev/null'
stateChangeUsesBody = false
```
It works in 2.1.10 but not in 2.1.12 or 2.1.13

**Comment by uglyog** [comment]
This should be fixed in 2.1.13 (issue #92). Can you check that the version of the gradle plugin is 2.1.13. I see that the gradle plugin portal is reporting 2.1.12 as the latest, but 2.1.13 is in maven central.

Looks like the publish process for the gradle plugin portal has changed. I'll update it to get 2.1.13 to be the latest.

**Comment by uglyog** [comment]
Finally managed to get 2.1.13 published to the plugin portal.
